netbox/utilities/templatetags/helpers.py

```diff
@@ -93,14 +93,29 @@ def humanize_speed(speed):
 def humanize_megabytes(mb):
     """
     Express a number of megabytes in the most suitable unit (e.g. gigabytes or terabytes).
+    It considers the mb value as megabytes and converts it to the most suitable unit.
     """
+
+    # Factors in bytes
+    factors = {
+        "mega": 1024 ** 2,
+        "giga": 1024 ** 3,
+        "tera": 1024 ** 4,
```

Comment:
I think we should change this logic to work on powers of 10 rather than powers of 2, given that the [commonly accepted definition of gigabyte](https://en.wikipedia.org/wiki/Gigabyte) is 1,000 megabytes.